cmd/edv-rest/startcmd/start.go

```diff
@@ -210,7 +252,7 @@ func createFlags(startCmd *cobra.Command) {
 	startCmd.Flags().StringP(tlsKeyFileFlagName, tlsKeyFileFlagShorthand, "", tlsKeyFileFlagUsage)
 }
 
-func startEDV(parameters *edvParameters) error {
+func startEDV(parameters *edvParameters, dbInit dbInitializer) error {
```

Comment:
(this is an example of somewhere where you would leave the acronym all uncapitalized (which you did))

Reply:
done!